GravityParticles.podspec

```diff
@@ -0,0 +1,22 @@
+
+Pod::Spec.new do |s|
+  s.name             = 'GravityParticles'
+  s.version          = '0.3.0'
+  s.source           = {
+    :git => 'https://github.com/buildit/gravity-particles.git',
+    :tag => s.version.to_s
+  }
+
+  s.summary          = 'GravityParticles to be used on iOS when styling your UIKit.'
+  s.description      = 'This library will contain basic swift files using style dictionnary from Amazon'
+  s.homepage         = 'https://github.com/buildit/gravity-particles'
+  s.license          = {
+    :type => 'MIT',
+    :file => 'LICENSE'
+  }
+  s.author           = {
+    'FiF0o' => 'jonathan.lazarini@wipro.com'
+  }
+  s.ios.deployment_target = '8.0'
+  s.source_files = 'Tokens/**/*'
```

Comment:
Hmm... there's a fair amount of overlap between the values here (version, description, etc.) and what's inside `package.json`. I'm wondering if we should have a little script (could be a Gulp task) that generates this podspec file.

That way the `package.json` becomes our central "source of truth" and this (and any similar files we might need in future) are auto-generated from that same info.

Reply:
yes, agreed however i would prefer using a shell script instead with some templating.
Perhaps we could use `sed` to find and replace some values as it would save us to install other dependencies and this probably could be achieved with one line of code :P
For example, something like that from npm would 
```
sed -r "s/(\'##version##\')/'$npm_package_version'/g" myPodspecFile.podspec
```
Or even get the current version with a regex?

Resolving as this has been move to the other branch @james-nash